src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/WindStaff.java

```diff
@@ -31,6 +31,11 @@ public ItemUseHandler getItemHandler() {
                 if (p.getInventory().getItemInMainHand().getType() != Material.SHEARS && p.getGameMode() != GameMode.CREATIVE) {
                     FoodLevelChangeEvent event = new FoodLevelChangeEvent(p, p.getFoodLevel() - 2);
                     Bukkit.getPluginManager().callEvent(event);
+                    
+                    if (event.isCancelled()) {
+                        SlimefunPlugin.getLocalization().sendMessage(p, "messages.wind-staff-fail", true);
+                        return;
+                    }
```

Comment:
I don't think we should send a message here, if the FoodLevelChangeEvent was cancelled, then we should simply not consume any hunger points. The wind staff itself did not fail, if the event was cancelled, this is most likely because that server doesn't use the hunger system...

Reply:
Yeah but still we need to feedback to the player as this is not lag or sth.